---
title: “Finally”不能出现在“Try”语句之外
ms.date: 07/20/2015
f1_keywords:
- vbc30382
- bc30382
helpviewer_keywords:
- BC30382
ms.assetid: 0314d8d2-18bc-4bbd-858c-0a18408b52fd
ms.openlocfilehash: 1d5987d6dc3b25088a55b6f0f43a8fb689f2c3e0
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: MT
ms.contentlocale: zh-CN
ms.lasthandoff: 04/23/2019
ms.locfileid: "62027519"
---
# <a name="finally-cannot-appear-outside-a-try-statement"></a>“Finally”不能出现在“Try”语句之外
`Finally` 用于完成 `Try...Catch...Finally` 块；因此它只能在块的末尾出现一次。 或者有不必要的 `Finally`，或者 `Finally` 语句出现在其对应 `Try` 块的边界之外。  
  
 **错误 ID:** BC30382  
  
## <a name="to-correct-this-error"></a>更正此错误  
  
1. 找到并删除不必要的 `Finally` 语句。  
  
2. 将 `Finally` 语句移到代码中的适当位置。  
  
## <a name="see-also"></a>请参阅

- [Try...Catch...Finally 语句](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
